k8s/logzio-deployment.yaml

```diff
@@ -0,0 +1,138 @@
+# serviceaccount
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: logzio-mysql-logs
+  namespace: kube-system
+  labels:
+    k8s-app: logzio-mysql-logs
+---
+# clusterrole
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: logzio-mysql-logs
+  labels:
+    k8s-app: logzio-mysql-logs
+rules:
+  - apiGroups:
+    - ""
+    resources:
+    - namespaces
+    - pods
+    - nodes
+    verbs:
+    - get
+    - watch
+    - list
+  - apiGroups:
+    - "apps"
+    resources:
+    - replicasets
+    verbs:
+    - get
+    - watch
+    - list
+---
+# clusterrolebinding
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: logzio-mysql-logs
+subjects:
+- kind: ServiceAccount
+  name: logzio-mysql-logs
+  namespace: kube-system
+roleRef:
+  kind: ClusterRole
+  name: logzio-mysql-logs
+  apiGroup: rbac.authorization.k8s.io
+---
+# daemonset
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: logzio-mysql-logs
+  namespace: kube-system
```

Comment:
why we deploy it on kube-system

Reply:
no special reason. You prefer it to be under "monitoring"?

Reply:
yes please